runtime/environment-cpu.yml

```diff
@@ -42,3 +42,6 @@ dependencies:
   - openslide-python==1.2.0
   - timm==0.4.12
   - opencv=4.7.0
+  - pip:
+    - staintools==2.1.2
+    - spams==2.6.5.4
```

Comment:
The staintools readme recommends conda instead of pip so let's move this outside of the pip block

Reply:
The conda repo has not been updated for three years, and this old version does not seem to be compatible with the current image. Isnt it possible to install it through PyPi ?